test/e2e/apimachinery/chunking.go

```diff
@@ -69,9 +68,9 @@ var _ = SIGDescribe("Servers with support for API chunking", func() {
 				if err == nil {
 					return
 				}
-				e2elog.Logf("Got an error creating template %d: %v", i, err)
+				framework.Logf("Got an error creating template %d: %v", i, err)
 			}
-			e2elog.Fail("Unable to create template %d, exiting", i)
+			framework.Fail("Unable to create template %d, exiting", i)
```

Comment:
/lgtm cancel
So it seems like there is no `framework.Fail()` method.
Could you please add this method to https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/log.go

Here is the defintion of `e2elog.Fail()`:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/log/logger.go#L57 to

Reply:
Thanks for pointing out, already updated.